src/csv/preprocessing.rs

```diff
@@ -23,6 +23,20 @@ pub enum Preprocessor {
     DeleteRowByNumber(usize),
     /// Replace all fields in row  where at least a single field matches regex by a deleted marker
     DeleteRowByRegex(String),
+    /// replace found cell using row and columnd index by a deleted marker
+    DeleteCellByNumber {
```

Comment:
This changes the json schema - please regenerate it via `cargo run -- schema > config_scheme.json` and also commit the result to this MR :)